shared/chat/conversation/header-area/normal/index.native.js

```diff
@@ -37,51 +29,47 @@ const ChannelHeader = ({
     >
       <Box style={{...globalStyles.flexBoxColumn}}>
         <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', alignSelf: 'center'}}>
-          <Avatar teamname={teamName} size={smallTeam ? 16 : 12} />
-          {!smallTeam && (
+          <Avatar teamname={props.teamName} size={props.smallTeam ? 16 : 12} />
+          {!props.smallTeam && (
             <Text type="BodySmallSemibold" style={{color: globalColors.black_40}}>
-              &nbsp;{teamName}
+              &nbsp;{props.teamName}
             </Text>
           )}
-          {smallTeam && (
+          {props.smallTeam && (
             <Text type="BodyBig" style={{color: globalColors.black_75}}>
-              &nbsp;{teamName}
+              &nbsp;{props.teamName}
             </Text>
           )}
-          {smallTeam && muted && <ShhIcon />}
+          {props.smallTeam && props.muted && <ShhIcon />}
         </Box>
-        {!smallTeam && (
+        {!props.smallTeam && (
           <Box style={{...globalStyles.flexBoxRow, alignSelf: 'center'}}>
             <Text type="BodyBig" style={{color: globalColors.black_75}}>
-              #{channelName}
+              #{props.channelName}
             </Text>
-            {muted && <ShhIcon />}
+            {props.muted && <ShhIcon />}
           </Box>
         )}
       </Box>
     </Box>
     <Icon
       type="iconfont-info"
       style={{...styleLeft, flexShrink: 0, padding: globalMargins.tiny, fontSize: 21}}
-      onClick={onToggleInfoPanel}
+      onClick={props.onToggleInfoPanel}
+    />
+    <Icon
+      type="iconfont-keybase"
+      style={{...styleLeft, flexShrink: 0, padding: globalMargins.tiny, fontSize: 21}}
+      onClick={props.onDebugDump}
     />
   </Box>
 )
 
-const UsernameHeader = ({
```

Comment:
this type of spreading-all-the-props just takes up a lot of space and doesn't really give us anything. if you want to pull something out thats fine but should only be a prop you reuse a lot in the render